templates/quickstart-crowd-dc.template.yaml

```diff
@@ -930,21 +903,13 @@ Mappings:
     us-gov-east-1:
       HVM64: ami-a2d938d3
 
-  JIRAProduct2NameAndVersion:
-    Core:
-      name: jira-core
-      shortdisplayname: '"Jira Core"'
-      fulldisplayname: '"Atlassian Jira Core"'
-    ServiceDesk:
-      name: servicedesk
-      shortdisplayname: '"Jira SD"'
-      fulldisplayname: '"Atlassian Jira Service Desk"'
-    Software:
-      name: jira-software
-      shortdisplayname: '"Jira SW"'
-      fulldisplayname: '"Atlassian Jira Software"'
+  CROWDProduct2NameAndVersion:
```

Comment:
As far as I know this is only required for Jira. We can probably drop this mapping and the corresponding variables in `/etc/atl`

Reply:
Done.